docs/supported_lockfiles.md

```diff
@@ -0,0 +1,35 @@
+# Supported Lockfiles
+
+The Phylum CLI supports processing many different lockfiles:
+
+| Lockfile type | Filenames                                                              |
```

Comment:
You renamed the other table from filenames to Manifests. Would make sense to rename this from filenames to Lockfiles then, right?

Reply:
Sure...it will be renamed.